src/js/charts/point.js

```diff
@@ -388,7 +374,8 @@ function mg_color_point_mouseover({color_accessor, scalefns}, elem, d) {
     size_domain: null,
     color_domain: null,
     active_point_size_increase: 1,
-    color_type: 'number' // can be either 'number' - the color scale is quantitative - or 'category' - the color scale is qualitative.
+    color_type: 'number', // can be either 'number' - the color scale is quantitative - or 'category' - the color scale is qualitative.
+    highlight: null // callback function to select the point to highlight
```

Comment:
let's change this comment to `// if this callback function returns true, the selected point will be highlighted`